docker/builders/builder-any-x86_64_gcc10.0.0_gcc9.0.0.Dockerfile

```diff
@@ -39,8 +39,10 @@ RUN apt-get update \
 	software-properties-common \
 	gpg \
 	zstd \
-    && rm -rf /var/lib/apt/lists/*
+	cmake \
+	git \
```

Comment:
We need `git` too.
We will run cmake configuration step in full mode, even if with lots of things disabled. Unfortunately, given that libs only [recently](https://github.com/falcosecurity/libs/pull/1325) gained the ability to build the bpf probe in MINIMAL_BUILD mode, we need to configure the project in full mode, and this requires `git` because grpc and protobuf deps need to be fetched/cloned.